test/test_ensemble_builder/test_ensemble.py

```diff
@@ -646,6 +646,42 @@ def test_read_pickle_read_preds(ensemble_backend):
     assert ensbuilder2.last_hash == ensbuilder.last_hash
 
 
+def testEnsembleSelection():
```

Comment:
Could you please move this into a file which is only testing ensemble selection?